wapitiCore/attack/mod_drupal_enum.py

```diff
@@ -30,13 +31,43 @@ def get_hashes(self):
             data = json.load(hashes)
             return data
 
+    @staticmethod
+    def calculate_git_hash(path):
+        try:
+            with open(path, 'rb') as file:
+                data = file.read()
+
+            # Calculate the size of the file
+            filesize = len(data)
+
+            # Create the string to hash to match the git hash function
```

Comment:
Please, add a comment to put a URL explaining the hash method, for example https://alblue.bandlem.com/2011/08/git-tip-of-week-objects.html might be handy

Reply:
solved